mis_builder/models/aep.py

```diff
@@ -88,7 +89,7 @@ class AccountingExpressionProcessor(object):
         r"\s*"
         r"(?P<account_sel>_[a-zA-Z0-9]+|\[.*?\])"
         r"\s*"
-        r"(?P<ml_domain>\[.*?\])?"
+        r"(?P<ml_domain>\[.*\])?"
```

Comment:
One thing that might then be slightly annoying is that the allowed syntax is not the same for the account and move line domains. To make it fully correct we'd need something more powerful than a regex.

Reply:
True, but that would be much of a bigger change